adadamp/_dist.py

```diff
@@ -57,6 +56,10 @@ def gradient(
     where `l` is the loss function for a single example.
 
     """
+    inputs = [pt[0] for pt in train_set] # client.scatter(train_data)
+    targets  = [pt[1] for pt in train_set]  #client.scatter(train_lbl)
```

Comment:
Caution: this does a lot of extra computation because `transform` is `transforms` is specified in `FashionMNIST`. That means anytime an object is produced in `train_set` the computation for `transform` is run. That means there are 50,000 transforms being done here.

I (very recently) made the same change #4: https://github.com/stsievert/adadamp/blob/e4a762efcd6438a873e236cd8dec119cff7ba9b4/adadamp/damping.py#L173-L175

 IIRC, before this change, computing the 128 transforms for a single batch took about 30% the time for a single step. Now it's halved and it takes 15% of the time.